elliottlib/cli/find_bugs_cli.py

```diff
@@ -220,16 +220,52 @@ def _filter_bugs(bugs):  # returns a list of bugs that should be processed
     # Otherwise we don't need to sweep bugs at all.
     if not (into_default_advisories or default_advisory_type):
         return
-    impetus_bugs = {}  # key is impetus ("rpm", "image", "extras"), value is a set of bug IDs.
-    # @lmeyer: simple and stupid would still be keeping the logic in python, possibly with config flags for branched logic. until that logic becomes too ugly to keep in python, i suppose..
+
+    # key is impetus ("rpm", "image", "extras"), value is a set of bug IDs.
+    impetus_bugs = {
+        "rpm": set(),
+        "image": set(),
+        "extras": set()
+    }
+
+    # @lmeyer: simple and stupid would still be keeping the logic in python,
+    # possibly with config flags for branched logic.
+    # until that logic becomes too ugly to keep in python, i suppose..
     if major_version < 4:  # for 3.x, all bugs should go to the rpm advisory
         impetus_bugs["rpm"] = set(bugs)
     else:  # for 4.x
-        # optional operators bugs should be swept to the "extras" advisory, while other bugs should be swept to "image" advisory.
-        # a way to identify operator-related bugs is by its "Component" value. temporarily hardcode here until we need to move it to ocp-build-data.
+        # sweep rpm cve trackers into "rpm" advisory
+        if mode == 'sweep' and cve_trackers:
+            rpm_bugs = bzutil.get_valid_rpm_cves(bugs)
+            green_prefix("RPM CVEs found: ")
+            click.echo([b.id for b in rpm_bugs])
+
+            # check if corresponding brew builds are attached to advisory
+            attached_builds = errata.get_advisory_nvrs(runtime.group_config.advisories["rpm"])
+            packages = attached_builds.keys()
+            failed = []
+            for bug, package_name in rpm_bugs.items():
+                if package_name not in packages:
+                    failed.append((bug.id, package_name))
```

Comment:
CVEs not belonging to rpm advisory should go to the main image or the extras advisory, i guess.

Reply:
They would be at line 268..